rainbow.xcodeproj/project.pbxproj

```diff
@@ -480,6 +486,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = 8T393AQ955;
```

Comment:
Fine with me for now, but we should really try to push on getting the account we're going to publish this with decided on soon. I know @antonmc used his account for Kubecoin, so should we just do that?